examples/scaling-criteo/02-ETL-with-NVTabular.ipynb

```diff
@@ -110,6 +110,7 @@
     "\n",
```

Comment:
<p>this code uses <code>os.path.join</code> and assumes the pathsep is <code>/</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>